packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.html

```diff
@@ -27,4 +27,19 @@
             data-translate="dedicatedCloud_datacenters_adding_select"
         ></option>
     </select>
+
+    <div
+        class="mt-4"
+        data-ng-if="$ctrl.nsxStatus !== 'enabled' && $ctrl.commercialRange.model.name === 'premier'"
```

Comment:
Nitpick: "$ctrl.nsxStatus !== 'enabled' && $ctrl.commercialRange.model.name === 'premier'"
This line of code is used multiple times. Is it possible to defined it once, may be in a controller, and reuse it?

Reply:
I suggest this name :
`$ctrl.isValidNsxConfig()`
```JS
isValidNsxConfig() {
   return this.nsxStatus !== 'enabled' && this.commercialRange.model.name === 'premier'
}
```